yt_dlp/options.py

```diff
@@ -1245,7 +1245,7 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
     verbosity.add_option(
         '--console-title',
         action='store_true', dest='consoletitle', default=False,
-        help='Display progress in console titlebar')
+        help='Display progress in the console window\'s titlebar.')
```

Comment:
this was fine as-is imo
```suggestion
        help='Display progress in console titlebar.')
```